src/lib.rs

```diff
@@ -467,15 +478,18 @@ impl Client {
         nonce: Option<String>,
         signer: &impl Signer,
         url: &str,
-    ) -> Result<Response<Body>, Error> {
+    ) -> Result<Response<hyper::body::Incoming>, Error> {
         let nonce = self.nonce(nonce).await?;
         let body = JoseJson::new(payload, signer.header(Some(&nonce), url), signer)?;
         let request = Request::builder()
             .method(Method::POST)
             .uri(url)
             .header(CONTENT_TYPE, JOSE_JSON)
-            .body(Body::from(serde_json::to_vec(&body)?))
-            .unwrap();
+            .body(
+                http_body_util::Full::from(serde_json::to_vec(&body)?)
+                    .map_err::<_, Error>(|_| unreachable!("Should be Infallible"))
```

Comment:
I'd prefer to just stick with `unwrap()` instead of this `map_err()` contraption.

Reply:
Is there a way to cast `Body`'s associated `Error` type from `Infallible` to `instant_acme::Error` with `Result::unwrap()`? That's what's happening here.

Reply:
It's due to a change in how hyper works in 1.0